cluster/addons/calico-policy-controller/typha-horizontal-autoscaler-clusterrole.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
https://github.com/kubernetes/kubernetes/pull/54346

This related PR updates to use `v1` - we should probably do that here as well so we don't end up with a mix of versions.